src/Microsoft.DotNet.Interactive.Jupyter/JupyterInteractiveHost.cs

```diff
@@ -8,13 +8,9 @@
 
 namespace Microsoft.DotNet.Interactive.Jupyter
 {
-    public class JupyterInteractiveHost : IInteractiveHost
+    public static class JupyterInteractiveHost 
     {
-        public JupyterInteractiveHost()
-        {
-        }
-
-        public Task<string> GetInputAsync(string prompt = "", bool isPassword = false, CancellationToken cancellationToken = default)
+        public static  Task<string> GetInputAsync(string prompt = "", bool isPassword = false, CancellationToken cancellationToken = default)
```

Comment:
How will we enforce that this method signature is the same across hosts? Keep in mind notebooks need to be round tripped between VS Code, Jupyter, etc.

Reply:
that api is exposed in both vscode and jupyter mode

Reply:
Understood but what enforces that the API is the same for both?

Reply:
both experiences use the exact same GetInputAsync method

Reply:
sorry i see now what you mean